bin/porta

```diff
@@ -560,6 +582,7 @@ module Porta
         "PORT" => "3000",
         "CONFIG_INTERNAL_API_USER" => options[:internal_api_user],
         "CONFIG_INTERNAL_API_PASSWORD" => options[:internal_api_password],
+        "RAILS_ENV" => 'development',
```

Comment:
note to anybody that tries to revive this PR in the future:
this line prevents to override this value by setting `RAILS_ENV` in shell